arch/arm/core/aarch32/Kconfig

```diff
@@ -126,6 +126,13 @@ config COMPILER_ISA_THUMB2
 config NUM_IRQS
 	int
 
+config ARCH_SUPPORT_SHARED_IRQ
+	bool "Allow IRQs to share multiple ISRs"
+	default n
+	help
+	  Some ARM peripherals share an IRQ and this option enables the system
+	  to call all ISRs attached to a pending IRQ.
+
```

Comment:
This still needs to go in `arch/Kconfig` not here. Also, better using `ARCH_HAS`.

Reply:
I've moved option to arch/Kconfig and placed the option in the "Interrupt Configuration" section, named ARCH_SW_ISR_TABLE_SHARED_IRQ